lib/zclient.h

```diff
@@ -537,6 +555,16 @@ struct zapi_route {
  * offload situation.
  */
 #define ZEBRA_FLAG_OFFLOAD_FAILED     0x200
+/*
+ * This flag tells Zebra that the route is a seg6 route and should
+ * be treated specially.
+ */
+#define ZEBRA_FLAG_SEG6_ROUTE         0x400
```

Comment:
do these flags affect zebra's rib processing, or are they just about the contents of the zapi message? If they only are used in encode and decode, maybe they should be ZAPI_MESSAGE flags?

Reply:
@mjstapp 

You were right. I have always misunderstood the role of ZEBRA_FLAG_XX. And I should have used ZAPI_NEXTHOP_FLAG_XXX.

I have fixed it in the following commit.
Thanks for the great comments.

https://github.com/FRRouting/frr/pull/5865/commits/60566d3246758106a4fe2e066fc36eb9aa94742c